airflow/models/baseoperator.py

```diff
@@ -634,6 +651,33 @@ def deps(self) -> Set[BaseTIDep]:
             NotPreviouslySkippedDep(),
         }
 
+    def _resolve_dependants(self) -> None:
```

Comment:
Not 100% onboard with method name here. maybe better `_set_dependencies_xcomarg`?

Reply:
Sounds reasonable, changed to `_set_xcomargs_dependencies`